dockerfiles/init/manifests/codenvy.pp

```diff
@@ -288,6 +288,10 @@
   $puppet_src_folder = getValue("CHE_CONFIG","/path/to/codenvy/codenvy/puppet/sources")
 # path to codenvy tomcat for development mode
   $codenvy_development_tomcat = getValue("CHE_ASSEMBLY","/path/to/codenvy_tomcat")
+# path to codenvy ws agent for development mode
+  $codenvy_development_ws_agent = getValue("WS_AGENT_ASSEMBLY","/path/to/codenvy_tomcat")
+# path to codenvy terminal agent for development mode
+  $codenvy_development_terminal_agent = getValue("TERMINAL_AGENT_ASSEMBLY","/path/to/codenvy_tomcat")
```

Comment:
this is "default" path which never be used as we always set this in CLI but I agree i will update  this to be more clear, thanks